### DISCUSSION TITLE
Preventing Select After Save

### DISCUSSION THREAD

**Discussion by dlahn** [jeremyevans/sequel#2103] [General]
Is there a way to prevent the `SELECT` query from happening after calling `save` on a model?

Once we have saved the row we aren't interested in what comes back. I've had a look around but can't seem to find anything. I saw there was a plugin to turn it on for updates, but nothing to turn it off.

**Comment by jeremyevans** [comment]
I'm guessing you want the `skip_create_refresh` plugin.
